open-banking-accelerator/components/event-notifications/com.wso2.openbanking.accelerator.event.notifications.service/src/main/java/com/wso2/openbanking/accelerator/event/notifications/service/handler/DefaultEventSubscriptionServiceHandler.java

```diff
@@ -262,9 +281,10 @@ private EventSubscriptionResponse validateClientId(String clientId) {
         } catch (OBEventNotificationException e) {
             log.error("Invalid client ID", e);
             EventSubscriptionResponse eventSubscriptionResponse = new EventSubscriptionResponse();
-            eventSubscriptionResponse.setStatus(EventNotificationConstants.BAD_REQUEST);
-            eventSubscriptionResponse.setErrorResponse(String.format("A client was not found" +
-                    " for the client id : '%s' in the database.. ", clientId));
+            eventSubscriptionResponse.setStatus(EventNotificationConstants.BAD_REQUEST_CODE);
```

Comment:
Check here as well in accordance with [Comment](https://github.com/wso2/financial-open-banking/pull/3/files#r1312600945) above

Reply:
We need to change error message since accelerator level error message contains x-wso2-cient-id which is not available in UK

Reply:
Fixed in https://github.com/wso2/financial-open-banking/pull/3/commits/a31eef6176e5003bd359e183aae45a90a5874f2b